scripts/telemctl

```diff
@@ -95,14 +92,16 @@ telem_restart() {
 }
 
 usage() {
+  format='  %-10s %s\n'
+  printf "\n"
   printf "$SCRIPT - Control actions for telemetry services\n"
```

Comment:
Sorry, I should have caught this as well. I know it is not in your changes, but whilst you are fixing things....

If $SCRIPT contains formatting directives this will cause problems, and the correct thing is to use

printf "%s - Control actions ...\n" "$SCRIPT"